src/style.css

```diff
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  color: #3e4e61;
+}
+
+.container {
+  margin: 20px 0;
+}
+
+.container__search {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-weight: 300;
+}
+
+.container__search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.container__search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
Add ::before element to the input and set for it icon image as a background (now I don't see the image on focus state)